CMakeLists.txt

```diff
@@ -8,8 +8,14 @@ set (CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
 set(CMAKE_CONFIGURATION_TYPES "Debug;Release" CACHE STRING "" FORCE)
 
 include(BuildFlags)
+# Creates a multithreaded executable file. Different method to add the /MT and /MTd flags
+#add_compile_options($<$<CXX_COMPILER_ID:MSVC>:$<$<CONFIG:RELEASE>:/MT>$<$<CONFIG:DEBUG>:/MTd>>)
 #include(cotire)
```

Comment:
I think #include(cotire) removed unless someone has updated info.  From what I read, cotire is obsolete.  At cotire github from 3+ years ago --

> _The functionality provided by cotire has been superseded by features added to CMake 3.16. Support for pre-compiling and unity builds is now built into CMake. Thus, there will not be any further updates or support for this project._